extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -94,6 +97,66 @@ export function getAdNetworkConfig(type, autoAmpAdsElement) {
   return null;
 }
 
+/**
+ * A fake ad network integration that is mainly used for testing
+ * and demo purposes. This implementation gets stripped out in compiled
+ * production code.
+ * @implements {AdNetworkConfigDef}
+ * @visibleForTesting
+ */
+class PingNetworkConfig {
```

Comment:
Could you measure the size increase brought by the config? I wonder if we can skip the config in prod

Reply:
Discussed offline, currently we are not excluding specific blocks of code in PROD. And this change should be a small number of bytes